### PR TITLE
Astrato Analytics Integration and Data Integration

### PR DESCRIPTION
**Pull request by piersbatch**
Added listings in partner & data viz section for Astrato Analytics, along with required images.

## Checklist
- [x] Delete items not relevant to your PR
- [x] URL changes should add a redirect to the old URL via https://github.com/ClickHouse/clickhouse-docs/blob/main/docusaurus.config.js
- [x] If adding a new integration page, also add an entry to the integrations list here: https://github.com/ClickHouse/clickhouse-docs/blob/main/docs/en/integrations/index.mdx

### CODE REVIEW

**Comment by laeg** [comment]
@piersbatch was this tested locally?

Getting the following error when trying to render the PR:

```
Compiled with problems:X

ERROR in ./docs/en/integrations/data-visualization/astrato-and-clickhouse.md

Module build failed (from ./node_modules/@docusaurus/mdx-loader/lib/index.js):
SyntaxError: /Users/laeg/ClickHouse/clickhouse-docs/docs/en/integrations/data-visualization/astrato-and-clickhouse.md: Unexpected token, expected "," (32:127)

  30 | ​`}</li>
  31 | <li parentName="ul">{`Click on the `}<strong parentName="li">{`New Data Connection`}</strong>{` button in the top right side of the screen.`}
> 32 | <img  src={require('./images/astrato_1_dataconnection.png').default}  class="image"  alt="Astrato Data Connection"  style={{max-width:'50%',  'background-color':  'transparent'}}/>
     |                                                                                                                                ^
  33 | </li>
  34 | </ul>
  35 | <br/>
```

File: docs/en/integrations/data-visualization/astrato-and-clickhouse.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- Select **Data** in the sidebar, and select the **Data Connection** tab 
+(or, navigate to this link: https://app.astrato.io/data/sources)
+​
+- Click on the **New Data Connection** button in the top right side of the screen.
+<img  src={require('./images/astrato_1_dataconnection.png').default}  class="image"  alt="Astrato Data Connection"  style={{max-width:'50%',  'background-color':  'transparent'}}/>
+
+<br/>
+
+- Select **ClickHouse**. 
+<img  src={require('./images/astrato_2a_clickhouse_connection.png').default}  class="image"  alt="Astrato ClickHouse Data Connection"  style={{max-width:'50%',  'background-color':  'transparent'}}/>
```
Comment (laeg) [inline docs/en/integrations/data-visualization/astrato-and-clickhouse.md:44]: Need to be changed to `width`

File: docs/en/integrations/data-visualization/astrato-and-clickhouse.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+<ConnectionDetails />
+
+## Creating the data connection to ClickHouse
+
+- Select **Data** in the sidebar, and select the **Data Connection** tab 
+(or, navigate to this link: https://app.astrato.io/data/sources)
+​
+- Click on the **New Data Connection** button in the top right side of the screen.
+<img  src={require('./images/astrato_1_dataconnection.png').default}  class="image"  alt="Astrato Data Connection"  style={{max-width:'50%',  'background-color':  'transparent'}}/>
```
Comment (laeg) [inline docs/en/integrations/data-visualization/astrato-and-clickhouse.md:39]: needs to be changed to `width`

File: docs/en/integrations/data-visualization/astrato-and-clickhouse.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+In our Data View editor, you will see all of your Tables and Schemas in ClickHouse, select some to get started.
+
+<img  src={require('./images/astrato_4a_clickhouse_data_view.png').default}  class="image"  alt="Astrato connect to ClickHouse User Access"  style={{width:'50%',  'background-color':  'transparent'}}/>
+
+
+Now that you have your data selected, go to define the **data view**. Click define on the top right of the webpage.
+
+In here, you are able to join data, as well as, **create governed dimensions and measures** - ideal for driving consistency in business logic across various teams. 
+
+<img  src={require('./images/astrato_4b_clickhouse_data_view.png').default}  class="image"  alt="Astrato connect to ClickHouse User Access"  style={{width:'50%',  'background-color':  'transparent'}}/>
```
Comment (laeg) [inline docs/en/integrations/data-visualization/astrato-and-clickhouse.md:75]: file does not exist. 

should be: `docs/en/integrations/data-visualization/images/astrato_4b_clickhouse_data_view_joins.png`

File: docs/en/integrations/data-visualization/astrato-and-clickhouse.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+
+**Astrato intelligently suggests joins** using your meta data, including leveraging the keys in ClickHouse. Our suggested joins make it easy for you to gets started, working from your well-governed ClickHouse data, without reinventing the wheel. We also show you **join quality** so that you have the option to review all suggestions, in detail, from Astrato.
+
+<img  src={require('./images/astrato_4c_clickhouse_data_view.png').default}  class="image"  alt="Astrato connect to ClickHouse User Access"  style={{width:'50%',  'background-color':  'transparent'}}/>
```
Comment (laeg) [inline docs/en/integrations/data-visualization/astrato-and-clickhouse.md:80]: file does not exist.

should be: `docs/en/integrations/data-visualization/images/astrato_4c_clickhouse_completed_data_view.png`

**Review by laeg — CHANGES_REQUESTED** [review]
images refs and css styles need to be updated. 

Can you test these docs locally and upload a screenshot please when completed. Thanks

**Comment by piersbatch** [comment]
@laeg I'm back from annual leave and have done the changes, which have been tested locally.

**Comment by laeg** [comment]
cheers @piersbatch. can you rebase this branch(checks are saying there are conflicts) and also add yourselves to the table in [`data-visualisation.md`](https://github.com/ClickHouse/clickhouse-docs/pull/2719/files#diff-0fd8edb756195c56627c5e21083908d8fee12ae873906ee33d63ba42fcaede8eR44)

**Pull request closed without merging**